test/resources/custom-handlers/server.js

```diff
@@ -1,6 +1,4 @@
 const cds = require('@sap/cds')
-const path = require('path')
+// Programmatic configuration of GraphQL protocol adapter
```

Comment:
you're testing that this is possible?

Reply:
Yes, I wanted to leave in one test instance that uses programmatic configuration